command/agent/consul/script_test.go

```diff
@@ -0,0 +1,165 @@
+package consul
+
+import (
+	"context"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/nomad/nomad/structs"
+)
+
```

Comment:
Where are the normal case tests:
* Getting correct exit code
* Getting output
* Getting exec errors